app/controllers/verify/review_controller.rb

```diff
@@ -34,7 +34,7 @@ def create
     private
 
     def idv_profile_complete?
-      idv_session.resolution.try(:success?)
+      idv_session.profile_confirmation == true
```

Comment:
Understood.

Since your suggested approach discards the response from the `agent.start` method (the `Proofer::Resolution` object), then I guess it doesn't matter what this flag is called. I.e. `profile_confirmation` is a boolean, and `resolution` is a `Proofer::Resolution`. So your naming here is fine.